src/ophyd_async/core/_signal.py

```diff
@@ -412,6 +416,10 @@ async def observe_signals_value(
 ) -> AsyncGenerator[tuple[SignalR[SignalDatatypeT], SignalDatatypeT], None]:
     """Subscribe to the value of a signal so it can be iterated from.
```

Comment:
Nit: Outside scope I guess but:
```suggestion
    """Subscribe to a set of signals so they can be iterated from.
```